src/Inkluzitron/Services/PointsService.cs

```diff
@@ -47,6 +46,7 @@ public PointsService(DatabaseFactory factory, DiscordSocketClient discordClient,
             UsersService = usersService;
 
             DiscordClient.ReactionAdded += OnReactionAddedAsync;
+            DiscordClient.ReactionRemoved += OnReactionRemovedAsync;
```

Comment:
https://github.com/Inkluzitron/Inkluzitron/blob/master/src/Inkluzitron/Contracts/IReactionHandler.cs

Reply:
Tady si myslím že to není nutný. Nikdy to tady tu reakci neodstraní/nehodí `true` kdyby to bylo přes ten interface